scripts/split-e2e-tests.ts

```diff
@@ -224,7 +225,11 @@ const splitTests = (baseJobLinux: any, testDirectory: string, pickTests?: (testS
 
       if (RUN_SOLO.find((solo) => test === solo || test.match(solo))) {
         if (RUN_IN_ALL_REGIONS.find((allRegions) => test === allRegions || test.match(allRegions))) {
-          testRegions.forEach((region) => {
+          const shouldRunInNonOptInRegion = RUN_IN_NON_OPT_IN_REGIONS.find(
```

Comment:
What is this code used for? I cannot find the tests which meet all the three requirements (running solo, in all region and in non opt-in region)

Reply:
When I was generating the regionalized SQL canary tests, which meet this criteria I noticed this error and fixed it for future usage.